c++/src/kj/async.c++

```diff
@@ -829,11 +835,13 @@ struct FiberStack::Impl {
 
     // Note: mmap() allocates zero'd pages so we don't have to memset() anything here.
 
-    KJ_SYSCALL(getcontext(&impl.fiberContext));
-    impl.fiberContext.uc_stack.ss_size = allocSize - sizeof(Impl);
-    impl.fiberContext.uc_stack.ss_sp = reinterpret_cast<char*>(stack);
-    impl.fiberContext.uc_stack.ss_flags = 0;
-    impl.fiberContext.uc_link = &impl.originalContext;
+    KJ_SYSCALL(getcontext(&impl.fiberContext.context));
+    impl.fiberContext.context.uc_stack.ss_size = allocSize - sizeof(Impl);
+    impl.fiberContext.context.uc_stack.ss_sp = reinterpret_cast<char*>(stack);
+    impl.fiberContext.context.uc_stack.ss_flags = 0;
+     // to reuse fibers the start fibers runs forever, so this doesn't
```

Comment:
nitpick: Comment formatted weirdly. Extra space of indentation and wraps too early.

Reply:
The comment is also a little obtuse, maybe we could phrase it as: "We don't use `uc_link` because our fiber start routine runs forever in a loop to allow for reuse. When we're done with the fiber, we just destroy it, without switching back to its stack. This is safe since the start routine doesn't allocate any memory or RAII objects before looping."